examples/analytics.amp.html

```diff
@@ -127,6 +127,22 @@
 </script>
 </amp-analytics>
 
+<!-- Krux tracking. Ensure you replace CONFID with your configuration ID. -->
+<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/CONFID.json">
```

Comment:
Only that this vendor config won't change between clients (apart from triggers now), whereas I was trying to keep just the client specific config in the remote.  I don't have a very strong preference either way - let me know if you do and I'll respond accordingly.